Updated to resolve lint failure:

```
./charts/ceph-csi-rbd/values.yaml
  303:3     warning  comment not indented like content  (comments-indentation)
```

--- a/charts/ceph-csi-rbd/values.yaml
+++ b/charts/ceph-csi-rbd/values.yaml
@@ -304,9 +304,9 @@ topology:
   # for CSI nodeplugins to advertise their domains
   # NOTE: the value here serves as an example and needs to be
   # updated with node labels that define domains of interest
-  domainLabels:
-    - failure-domain/region
-    - failure-domain/zone
+  domainLabels: []
+  # - topology.kubernetes.io/region
+  # - topology.kubernetes.io/zone
 
 # readAffinity:
 # Enable read affinity for RBD volumes. Recommended to

